client/state/data-layer/wpcom/sites/simple-payments/index.js

```diff
@@ -63,35 +74,40 @@ export function customPostToProduct( product ) {
  * @returns { Object } custom post type data
  */
 export function productToCustomPost( product ) {
+	// Get the `product` object entries and filter only those that will go into metadata
+	const metadataEntries = Object.entries( product ).filter( ( [ key ] ) => metadataSchema[ key ] );
```

Comment:
`Object.entries` is [not supported](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#Browser_compatibility) in IE/Opera 🙁